packages/rpc-provider/src/http/polyfill.spec.ts

```diff
@@ -19,7 +22,7 @@ describe('http/polyfill', () => {
   });
 
   it('polyfills with no exceptions (with fetch)', () => {
-    global.fetch = () => true;
+    expect(global.fetch).toBeTruthy();
```

Comment:
No: we should keep `global.fetch = () => true;` here, or else this test is the same as the one before